<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>462</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>160000</width>
    <height>160000</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1024</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainServer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMenuBar {
	color:rgb(223,219,210);
	background-color:rgb(52, 101, 164);
}
QMenuBar::item {
	padding-top:4px;
	padding-left:4px;
	padding-right:4px;
	color:rgb(223,219,210);
	background-color:rgb(52, 101, 164);
}
QMenuBar::item:selected {
	color:rgb(255,255,255);
	padding-top:2px;
	padding-left:2px;
	padding-right:2px;
	border-top-width:2px;
	border-left-width:2px;
	border-right-width:2px;
	border-top-right-radius:4px;
	border-top-left-radius:4px;
	border-style:solid;
//	background-color:rgb(65,64,59);
    background-color:rgb(52, 101, 164);
	border-top-color: rgb(47,47,44);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(90, 87, 78, 255), stop:1 rgba(47,47,44, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(90, 87, 78, 255), stop:1 rgba(47,47,44, 255));
}
QMenu {
	color:rgb(223,219,210);
	background-color:rgb(65,64,59);
}
QMenu::item {
	color:rgb(223,219,210);
	padding-left:20px;
	padding-top:4px;
	padding-bottom:4px;
	padding-right:10px;
}
QMenu::item:selected {
	color:rgb(255,255,255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(225, 108, 54, 255), stop:1 rgba(246, 134, 86, 255));
	border-style:solid;
	border-width:3px;
	padding-left:17px;
	padding-top:4px;
	padding-bottom:4px;
	padding-right:7px;
	border-bottom-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(175,85,48,255), stop:1 rgba(236,114,67, 255));
	border-top-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
	border-right-color:qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
	border-left-color:qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>0</y>
      <width>26</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>esc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRefresh">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>551</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>http://localhost/index.html</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGo">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QWebEngineView" name="m_webView" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>931</width>
      <height>401</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>160000</width>
      <height>160000</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>1024</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <zorder>toolButton</zorder>
   <zorder>btnRefresh</zorder>
   <zorder>btnGo</zorder>
   <zorder>lineEdit</zorder>
   <zorder>m_webView</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Configure</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionCapture"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionTestButton"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionadd">
   <property name="text">
    <string>add</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionWebMode">
   <property name="text">
    <string>WebMode</string>
   </property>
  </action>
  <action name="actionCapture">
   <property name="text">
    <string>Capture</string>
   </property>
  </action>
  <action name="actionTestButton">
   <property name="text">
    <string>TestButton</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
